test/nools/task-emitter.spec.js

```diff
@@ -95,6 +95,206 @@ describe('task-emitter', () => {
         });
       });
 
+      it('task priority is not required', () => {
+        // given
+        const contact = personWithReports(aReport());
+        const task = aPersonBasedTask();
+        const config = {
+          c: contact,
+          targets: [],
+          tasks: [ task ],
+        };
+        delete config.tasks[0].priority;
+        
+        // when
+        const { emitted } = runNoolsLib(config);
+
+        // then
+        expect(emitted).to.have.length(2);
+        expect(emitted[0]).to.be.shallowDeepEqual({
+          '_type': 'task',
+          'actions': [{
+            'content': {
+              'contact': {
+                '_id': contact.contact._id,
+                'reported_date': TEST_DATE,
+                'type': 'person'
+              },
+              'source': 'task',
+              'source_id': contact.contact._id
+            },
+            'form': 'example-form',
+            'label': 'Follow up',
+            'type': 'report'
+          }],
+          'contact': {
+            '_id': contact.contact._id,
+            'reported_date': TEST_DATE,
+            'type': 'person'
+          },
+          'date': TEST_DAY,
+          'resolved': false
+        });
```

Comment:
I believe that the assertion here would work even if the task ended up having a `priority` property. Can you please add an additional assertion to check that indeed the task ends up with no `priority`. Something like: 

```
expect(emitted[0]).to.not.have.property('priority');
```